src/api/app/models/issue_tracker.rb

```diff
@@ -32,6 +33,18 @@ def write_to_backend
     self.issues_updated ||= Time.now
   end
 
+  def cve?
+    kind == "cve"
+  end
+
+  def verify_issue_name!(name)
```

Comment:
If we need 3 methods to do it right then is ok. Having things in the right place make code simpler, not more complicated. 

So, please, put the method in the Issue class that it is supposed to belong to. Pass the tracker as a param to it, break the conditions into understandable one line boolean expressions assigned to good named variables. And also handle the exception outside the method, we need to change this way of doing things, I beg you.